core/src/main/scala/com/criteo/cuttle/Metrics.scala

```diff
@@ -0,0 +1,22 @@
+package com.criteo.cuttle
```

Comment:
Can we scope all this into a `Metrics` object, so in the public API it's all coherent?